app/components/search_tips_link_component.html.erb

```diff
@@ -1,4 +1,4 @@
-<div class="align-self-center searchtips-link ms-lg-2 mt-2 mt-lg-0">
+<div class="<%= @classes %> searchtips-link ms-lg-2 mt-2 mt-lg-0">
```

Comment:
This probably isn't a big issue, but it could be confusing to have some classes set in the template and others set by the initialize classes argument. Maybe it's okay to leave the `searchtips-link` class set in the template, but set the bootstrap classes as default arguments to initialize?